rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/db/RMDBManagerBuffer.java

```diff
@@ -145,16 +146,18 @@ protected void removeKnownNodeSourceAndPendingUpdates(final String nodeSourceNam
         }
     }
 
-    protected void addUpdateNodeSourceToPendingDatabaseOperations(final NodeSourceData nodeSource) {
+    void addUpdateNodeSourceToPendingDatabaseOperations(final NodeSourceData nodeSource) {
         pendingNodeSourceUpdatesLock.lock();
         try {
             String nodeSourceName = nodeSource.getName();
             if (knownNodeSources.contains(nodeSourceName)) {
                 cancelScheduledNodeSourceTransaction();
                 pendingNodeSourceUpdates.put(nodeSourceName, nodeSource);
                 if (delayEqualsToZero) {
+                    logger.debug("Apply update node source " + nodeSource.getName() + " in database with no delay");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " in database with no delay" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)